.github/workflows/update-gh-pages.yml

```diff
@@ -10,6 +10,147 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Install Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.13'
+
+    - name: Install dependencies
+      shell: bash
+      run: |
+
+        python -m pip install --upgrade pip
+        pip install requests
+
+    - name: Write script
+      shell: bash
+      run: |
+
+        cat > /tmp/catsitemap.py << "EOF"
+        import re
+        import os
+        import xml.etree.ElementTree as ET
+        import requests
+        from xml.dom import minidom
```

Comment:
Also, I see there is a `LINKS` variable containing links to all documentation. This could allow us to simplify the regex part if we import that variable into your script, @moe-ad, see https://github.com/ansys/pyansys/blob/c3713e144c950bf87f52958a9ed971ff2747f4cc/tools/links.py#L23